css/style.css

```diff
@@ -231,3 +231,97 @@ footer h5 {
 .p3-text p {
   font-style: italic;
 }
+
+.main2 {
+  display: flex;
+  flex-direction: row;
+  flex-wrap: wrap;
+}
+
+form {
+  display: flex;
+  flex-direction: column;
+  gap: 20px;
+}
+
+.section2 {
```

Comment:
This is working fine, but your selector which targets all `section` elements is adding margin-top that you don't need